### PR TITLE
Update vendor packages

### PR DESCRIPTION
**Pull request by rf232**
Updated the vendor packages + resolved update in our sources

also wrote a quick howto for the next person who wants to do this

### CODE REVIEW

**Comment by rf232** [comment]
PTAL, mostly interesting are the second and third commit, first one is all govendor generated content

File: docs/devel/updating-deps.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+After having struggled getting the vendor packages upgraded I decided to give a
```
Comment (bryk) [inline docs/devel/updating-deps.md]: I guess we can remove this "after having struggled" section :) Docs should be emotion-less.

File: docs/devel/updating-deps.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+short description on how I managed to upgrade the vendor packages. There is no
+guarantee that this is the easiest/best way to do this, but it a way
```
Comment (bryk) [inline docs/devel/updating-deps.md]: s/but it a way/it is a way/

File: docs/devel/updating-deps.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Introduction
```
Comment (bryk) [inline docs/devel/updating-deps.md]: Add a header or paragraph that this is about updating godeps

File: docs/devel/updating-deps.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+After having struggled getting the vendor packages upgraded I decided to give a
+short description on how I managed to upgrade the vendor packages. There is no
+guarantee that this is the easiest/best way to do this, but it a way
+
+# Step by step guide
+
+- Have the dashboard source checked out in `${GOROOT}/src/github.com/kubernetes/dashboard`
```
Comment (bryk) [inline docs/devel/updating-deps.md]: GOROOT or GOPATH ?

**Comment by rf232** [comment]
PTAL, minor updates to doc, also plz merge for me if it is good, will be limited connectivity for me today

@bryk

**Comment by bryk** [comment]
LGTM. I'll merge this soon :)

**Pull request merged**